agent/util-scripts/pbench-register-tool-trigger

```diff
@@ -19,17 +20,24 @@ stop_trigger=""
 
 function usage() {
 	printf "usage:\n"
-	printf "$script_name --group=<tool group> --start-trigger=<string to match> --stop-trigger=<string to match>"
+	printf "$script_name [--group=<tool group>] --start-trigger=<string to match> --stop-trigger=<string to match>\n"
 }
 
+case $# in
+    0)
+        usage
```

Comment:
Perhaps `usage >&2`?

Reply:
Fixed (here and elsewhere - I hope I got them all).